notebooks/json_blocks/WET/BLOCK-T413.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ranges = [150, 5000, 5000, 0.1, 0.1]
</pre><p>These ranges and test patterns should be added somewhere in Zephyr scale for reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>